src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6303,3 +6303,29 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_mish_activation) {
 
     test_case.run_with_tolerance_as_fp(0.000001f);
 }
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_multinomial_13_param_inputs) {
+    auto model = convert_model("multinomial.onnx");
+
+    auto test_case = ov::test::TestCase(model, s_device);
+
+    test_case.add_input<float>({0.1f, 0.2f, 0.3f, 0.4f});
+    test_case.add_input<int>({3});
+
+    test_case.add_expected_output<int>({1, 0, 2, 1});
+
+    test_case.run();
+}
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_multinomial_13_const_inputs) {
+    auto model = convert_model("multinomial.onnx");
+
+    auto test_case = ov::test::TestCase(model, s_device);
+
+    test_case.add_input<float>({0.4f, 0.5f, 0.1f, 0.3f, 0.2f, 0.5f});
```

Comment:
As I understand, sum of all values in a single batch should be equal to 1.0f. Anyway, it isn't looks like {1, 10}, so it is definitely looks like an issue.

Reply:
I agree, these values can be a combination of any floating point values `unnormalized log probabilities`, tests should cover that case

Reply:
I suggest to do like others - check only output shape, but it requires to slightly change input values to have a shape slightly more complicated